<launch>
  <!--<param name="/use_sim_time" value="True" />-->
  <!-- Run VO -->
  <group ns="wide_stereo">
    <node name="vo" pkg="imapping" type="vo" output="screen" cwd="node">
      <remap from="left/image_rect" to="/wide_stereo/left/image_rect" />
      <remap from="left/camera_info" to="/wide_stereo/left/camera_info" />
      <remap from="right/image_rect" to="/wide_stereo/right/image_rect" />
      <remap from="right/camera_info" to="/wide_stereo/right/camera_info" />
    </node>
  </group>
</launch>

